sections/featured-product-custom.liquid

```diff
@@ -0,0 +1,171 @@
+{{ 'custom.css' | asset_url | stylesheet_tag }}
+<script src="{{ 'custom.js' | asset_url }}" defer></script>
+
+<div class="featured-product">
+    <div class="featured-product__container column-2">
+        {%-for block in section.blocks-%}
+            <div class="featured-product__img-container">
+                <picture>
+                    <source media="(min-width: 992px)" srcset="{{ block.settings.image | img_url: 'large' }}">
+                    <source media="(min-width: 768px)" srcset="{{ block.settings.image_2 | img_url: 'medium' }}">
+                    <source media="(max-width: 767.87px)" srcset="{{ block.settings.image_3 | img_url: 'medium' }}">
+                    <img class="featured-product__img" src="{{ block.settings.image | img_url: 'large' }}">
```

Comment:
no need to upload same image 3 times if the req is the same image. just use the same image with different img_url filter (large, medium, small, master, etc)